secp256k1Native/src/main/java/org/bitcoin/NativeSecp256k1.java

```diff
@@ -0,0 +1,446 @@
+/*
```

Comment:
Are these files not part of the submodule repo? 

If you are just trying to move them into the "right" place for sbt, I would consider keeping them in their current location and changing where sbt looks for them.  This approach seems brittle in the face of upstream changes.

Reply:
Not exactly the same as original ones, I added few lines that should take care of loading proper binaries depending on the OS.